### PR TITLE
create dynamodb table if it does not exists

### PR DESCRIPTION
**Pull request by motatoes**
DynamoDB table is not being created during the first run as mentioned in #111 resulting in `ResourceNotFound` Exception. This Pull request creates the table during any of the locking operations if it does not yet exist already. 

TODO: 

- Make the table creation only the first time (maybe using a flag of some sort?) Since otherwise there will be too much AWS calls happening for every action to check if the table exists.
- Write additional tests for this 
- Ensure there is no race conditions for the first run since this often occurs due to eventual consistency of AWS

### CODE REVIEW

File: pkg/aws/dynamo_locking.go [hunk 1]
```diff
@@ -14,7 +16,39 @@ type DynamoDbLock struct {
 	DynamoDb *dynamodb.DynamoDB
 }
 
+func (dynamoDbLock *DynamoDbLock) createTableIfNotExists() {
+	input := &dynamodb.CreateTableInput{
```
Comment (veziak) [inline pkg/aws/dynamo_locking.go]: would it be better if we check if table exist explicitly?

Comment (veziak) [inline pkg/aws/dynamo_locking.go]: would be good to log something if table has been created

File: pkg/aws/dynamo_locking.go [hunk 2]
```diff
@@ -14,7 +16,39 @@ type DynamoDbLock struct {
 	DynamoDb *dynamodb.DynamoDB
 }
 
+func (dynamoDbLock *DynamoDbLock) createTableIfNotExists() {
```
Comment (Spartakovic) [inline pkg/aws/dynamo_locking.go:19]: I think it's misleading to call it createTableIfNotExists() given it doesn't check that, might as well just call it createTable()

**Review by veziak — APPROVED** [review]
lgtm

**Pull request merged**